src/modules/Nsswitch.rb

```diff
@@ -53,7 +53,12 @@ def ReadDb(db_name)
     # @param db_name [String] database entry name, e.g. "passwd"
     # @param services [Array<String>] service specs, e.g. ["files", "nis"]
     def WriteDb(db_name, services)
-      cfa_model.update_entry(db_name, services)
+      # For improved compatibility with the old implementation, check also [""]
```

Comment:
maybe I would just add note to yardoc that passing [""] is deprecated

Reply:
I don't think it was never documented as a supported value for deleting entries. I just realized it was a (likely unintended) possibility by reading the old implementation... and I decided to implement some compatibility because it wouldn't be the first time we find code that relies on some undocumented internal.